src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedInstanceMethodSymbol.cs

```diff
@@ -83,4 +68,25 @@ internal override int CalculateLocalSyntaxOffset(int localPosition, SyntaxTree l
 
         internal sealed override bool UseUpdatedEscapeRules => ContainingModule.UseUpdatedEscapeRules;
     }
+
+    /// <summary>
+    /// A base class for synthesized methods that want a this parameter.
+    /// </summary>
+    internal abstract class SynthesizedInstanceMethodSymbol : SynthesizedMethodSymbol
+    {
+        private ParameterSymbol _lazyThisParameter;
+
+        public override bool IsStatic => false;
```

Comment:
>override [](http://example.com/codeflow?start=15&length=8)

sealed?

Reply:
This had other ramifications which I'd rather not chase for this PR. Plus this is existing code.